docs/index.md

```diff
@@ -67,12 +66,12 @@ The easiest way to get started with Swarm is to use the
 [official Docker image](https://registry.hub.docker.com/_/swarm/).
 
 ```bash
-docker pull swarm
+$ docker pull swarm
```

Comment:
I updated according to your suggestion but think this needs to be clarified somewhere. :)